eo-maven-plugin/src/main/java/org/eolang/maven/optimization/OptCached.java

```diff
@@ -41,9 +41,12 @@
  * Returns already optimized XML if it's found in the cache.
  *
  * @since 0.28.11
- * @todo #2674:30min The function {@code  OptCached.contains(final XML xml)}
- *  isn't work properly. This function compares caching and compilation times,
- *  which may lead to erroneous results. We need to fix this.
+ * @todo #2746:30min The tests
+ *  {@link org.eolang.maven.optimization.OptCachedTest#returnsFromCacheCorrectProgram(Path path)}
```

Comment:
@Yanich96 What do you think about the following description?
```java
// @todo #2746: 30min Use checksum, not time.
//  The following tests show that fetching from the cache doesn't work correctly:
//   - {@link org.eolang.maven.optimization.OptCachedTest#returnsFromCacheCorrectProgram(Path path)}
//   - {@link org.eolang.maven.optimization.OptCachedTest#returnsFromCacheButTimesSaveAndExecuteDifferent(Path path)}
//  Need to fix file validation from the cache: using checksum, but not time.
//  Don't forget to enable the tests.
```
Fixed grammar mistakes and the name of the test.